pages/vi/profiles/kholood-ea.md

```diff
@@ -0,0 +1,23 @@
+ #  Kholood Eassa <br>
+ 
+ ![Egypt](https://media.webjet.com.au/Images/cities/1200x472/CAI1.jpg)
+
+Location/Time zone : Egypt, cairo, GMT+2 <br>
```

Comment:
There should not be need of <br> It is HTML tag try to minimize or eliminate HTML tags as much as possible